tests/foreman/ui/test_discoveryrule.py

```diff
@@ -87,86 +89,68 @@ def gen_int32(min_value=1):
 
 
 @pytest.mark.tier2
-def test_positive_create_rule_with_non_admin_user(manager_loc, manager_user, module_org, test_name):
-    """Create rule with non-admin user by associating discovery_manager role
+def test_positive_crud_with_non_admin_user(
+    manager_loc, manager_user, module_org, test_name, module_target_sat
```

Comment:
Can we use module_location here instead of manager_loc?

Reply:
updated